events/dvg_lotharingia.txt

```diff
@@ -0,0 +1,1066 @@
+﻿namespace = dvg_lotharingia
+
+
+
+dvg_lotharingia.1 = {
+	type = country_event
+	placement = ROOT
+
+	title = dvg_lotharingia.1.t
+	desc = dvg_lotharingia.1.d
+	flavor = dvg_lotharingia.1.f
+	
+	event_image = {
+		video = "gfx/event_pictures/middleeast_courtroom_upheaval.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/middleeast/courtroom_upheaval"
+
+	icon = "gfx/interface/icons/event_icons/event_fire.dds"
+
+	duration = 3
+
+	trigger = {
+	    year = 1836
+	}
+	
+	
+	# Vive!
+	option = {
+		name = dvg_lotharingia.1.a
+		default_option = yes
+		add_journal_entry = {
+			type = je_lotharian_house_of_cards
+		}
+		add_radicals = {
+			value = 0.4
+			culture = cu:lotharian
+		}
+		add_loyalists = {
+			value = 0.15
+			culture = cu:lotharian
+		}
+		ai_chance = {
+			base = 100
+		}
+	}
+}
+
+
+
+dvg_lotharingia.2 = {
+	type = country_event
+	placement = ROOT
+
+	title = dvg_lotharingia.2.t
+	desc = dvg_lotharingia.2.d
+	flavor = dvg_lotharingia.2.f
+	
+	event_image = {
+		video = "gfx/event_pictures/unspecific_sick_in_hospital.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/middleeast/courtroom_upheaval"
+
+	icon = "gfx/interface/icons/event_icons/event_fire.dds"
+
+	duration = 3
+
+	trigger = { 
+	}
+	
+	
+	# Liberte!
+	option = {
+		name = dvg_lotharingia.2.a
+		default_option = yes
+		country_prestige_mult = -0.01

```

Comment:
Unknown effect country_prestige_mult at multiple places.
This does not work. You need to use static modifiers to apply prestige.